plugins/src/gcc/imp.rs

```diff
@@ -0,0 +1,1270 @@
+// Implements https://datatracker.ietf.org/doc/html/draft-ietf-rmcat-gcc-02
```

Comment:
```suggestion
///! Implements https://datatracker.ietf.org/doc/html/draft-ietf-rmcat-gcc-02
```

Reply:
What is the `!` for?

Reply:
With `!` it applies to the surrounding item (in this case the whole module), without to the following item (e.g. the next struct definition). It's the same thing with attributes, e.g.

```rust
// Applies to everything in this module
#![allow(dead_code)]
struct Foo {
    ...
}

// Applies only to the struct
#[allow(clippy::blabla)]
struct Bar {
    ...
}
```